jest.config.ts

```diff
@@ -0,0 +1,78 @@
+import { JestConfigWithTsJest } from "ts-jest";
+
+const baseConfig: JestConfigWithTsJest = {
+  roots: ["<rootDir>"],
+  clearMocks: true,
+  testMatch: ["**/(__tests__|src)/**/?(*.)+(spec|test).+(ts|tsx|js)"],
```

Comment:
We should really standardise around either `.test.ts` or `.spec.ts`, but not both. Maybe we can already change this to: 
```suggestion
  testMatch: ["src/**/(.+).spec.ts"],
```

Elsewhere we use: 
```
testRegex: "/src/.*\\.test\\.ts$",
```

This is definitely a future problem to tackle though. But lets standardise across all our projects.

Reply:
This repo should consistently be using `.spec.ts`, and I have no problem changing to `.test.ts`, just in another PR :)